app/assets/javascripts/angular/controllers/graph_ctrl.js

```diff
@@ -135,6 +138,9 @@ angular.module("Prometheus.controllers").controller('GraphCtrl',
   }
 
   $scope.refreshGraph();
+
+  $scope.drag = new DragSelection();
```

Comment:
this doesn't need to be created here; move it into the graph_chart and instantiate it there. also move the `setup()` function to the instantiation.